### PR TITLE
add description for usbbackup module

### PR DESCRIPTION
**Pull request by mi-hol**
Try to perform outlined task

- [ ] References issue
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: deb/openmediavault-usbbackup/usr/share/openmediavault/engined/module/usbbackup.inc [hunk 1]
```diff
@@ -25,6 +25,10 @@ class UsbBackup extends \OMV\Engine\Module\ServiceAbstract
 		return "usbbackup";
 	}
 
+	public function getDescription() {
+		return gettext("Backup to USB device");
```
[the remark this answers is not in the document]
Comment (mi-hol) [reply deb/openmediavault-usbbackup/usr/share/openmediavault/engined/module/usbbackup.inc:29]: Well I tried to be helpful to the best of my knowledge. I'm unaware of documentation for the plug-in.
The comment isn't fully clear to me.
Is your suggestion on how the text should read "syncs data to and from USB devices"?

[the remark this answers is not in the document]
Comment (votdev) [reply deb/openmediavault-usbbackup/usr/share/openmediavault/engined/module/usbbackup.inc:29]: You can have a look into the package description to get some ideas, e.g. https://github.com/openmediavault/openmediavault/blob/master/deb/openmediavault-usbbackup/debian/control#L14. This is also what i suggest when adding short descriptions for other modules. Looking into the Debian packages at https://packages.debian.org may help to get some ideas.

[the remark this answers is not in the document]
Comment (mi-hol) [reply deb/openmediavault-usbbackup/usr/share/openmediavault/engined/module/usbbackup.inc:29]: > what i suggest when adding short descriptions for other modules. Looking into the Debian packages at https://packages.debian.org may help to get some ideas.

sounds like a good specification of task :)
Will do.

**Comment by mi-hol** [comment]
> The PR title should be descriptive.

Guessing what would be descriptive for a receiver is a challenge, suggestions are therefore a good approach to bridge a gap.

File: deb/openmediavault-usbbackup/usr/share/openmediavault/engined/module/usbbackup.inc (hunk 1 above)
[the remark this answers is not in the document]
Comment (mi-hol) [reply deb/openmediavault-usbbackup/usr/share/openmediavault/engined/module/usbbackup.inc]: updated to same text as in https://github.com/openmediavault/openmediavault/blob/master/deb/openmediavault-usbbackup/debian/control#L14.

**Comment by mi-hol** [comment]
Signed-off-by: mi-hol 22799428+mi-hol@users.noreply.github.com

**Comment by votdev** [comment]
> Signed-off-by: mi-hol [22799428+mi-hol@users.noreply.github.com](mailto:22799428+mi-hol@users.noreply.github.com)

You need to add the DCO to the commit message(s) to make the check happy. It is not enough to add it at PR level.

**Comment by votdev** [comment]
Please squash your commits. Two commits for a one-liner is unnecessary and pollutes the commit history.

**Comment by mi-hol** [comment]
> Please squash your commits. Two commits for a one-liner is unnecessary and pollutes the commit history.

Apologize, we had the same issue already in https://github.com/openmediavault/openmediavault/pull/892#issuecomment-753470944
Since then I stopped using a local repository copy and am only using GitHub web-based GUI.
Unfortunately I don't see an option to squash a commit in my forked repository.
If I understand [GitHub's Doc 'Require linear history'](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-linear-history) correctly, this is a setting to be made in the target repository.
Appreciate any advice on how to avoid this issue from re-occurring.

**Comment by mi-hol** [comment]
> If I understand [GitHub's Doc 'Require linear history'](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-linear-history) correctly, this is a setting to be made in the target repository.
> Appreciate any advice on how to avoid this issue from re-occurring.

was my understanding wrong?

**Comment by votdev** [comment]
@mi-hol Your support in honor, but if I have to take care of everything myself then I might as well do it myself since it doesn't take any work away from me. I can't and won't bother that contributors understand how git works. This is a requirement to contribute to an OSS project. Maybe you should switch to a gui based tool like [SmartGit](https://www.syntevo.com/smartgit/) to help you with your work.

Please don't forget that commits need a signed-by for the checks ([DCO](https://probot.github.io/apps/dco/)) to be successful.

**Comment by mi-hol** [comment]
> take any work away from me

well the proposed repository setting change would take away that work once and forever. 
I'm sorry but fail to understand why there is arguing about a simple to implement win-win change that simplifies the workflow for all involved parties :(

**Comment by mi-hol** [comment]
As there is no interest to lower the burden for maintainer **and** contributors via simple repository setting change described above, I'm closing the PR

**Comment by votdev** [comment]
> > take any work away from me
> 
> well the proposed repository setting change would take away that work once and forever. I'm sorry but fail to understand why there is arguing about a simple to implement win-win change that simplifies the workflow for all involved parties :( To be clear https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests is what I suggest as change.

I want to have simple PRs with only one commit. I don’t want to let GitHub do any magical things that may break or produce incorrect results in bad situations.

It should not be a big burden to the contributors to squash their commits before submitting a PR to fulfill the requirements.

**Comment by mi-hol** [comment]
> I don’t want to let GitHub do any magical things that may break or produce incorrect results in bad situations.

This statement about GitHub magic is completely incorrect!
Like OMV is a wrapper over a number of tools, GitHub is a wrapper over git.
As soon as a repo is correctly configured git's "squash your commit" command is used by GitHub.
example from another rep:
2023-01-17 10_16_52-fix README content and links by mi-hol · Pull Request #2 · mi-hol_rusefi_documen

**Comment by votdev** [comment]
@mi-hol To make it clear, i don't want to discuss this. I want clear PR's, dot.

**Comment by mi-hol** [comment]
Sure you can insist on "everybody has to do it my way" but as a consequence please stop complaining about "there is no community engagement and only dev"

**Comment by ryecoaaron** [comment]
People are not avoiding PRs because they have to submit a PR signed in one commit.

**Pull request closed without merging**